cron.yaml

```diff
@@ -22,4 +22,7 @@ cron:
       schedule: every 15 minutes from 00:08 to 23:58
   -   description: "check app health status"
       url: /_ah/health
-      schedule: every 1 minutes
\ No newline at end of file
+      schedule: every 1 minutes
+  -   description: "check immutable sink data"
+      url: /api/tasks/check_immutable_backups
+      schedule: every 10 minutes from 00:12 to 23:52
```

Comment:
Finding a "correct" or "right" frequency is a task in it self. But regarding that IAM actually logs permission changes for auditing which tackles **security** this cronjob only check for **compliance**. thus i would suggest changing this to 1h

Reply:
ping change schedule to every hour